pkg/gatherers/clusterconfig/gather_monitoring_pvs_test.go

```diff
@@ -0,0 +1,41 @@
+package clusterconfig
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kubefake "k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_GatherMonitoring(t *testing.T) {
+
+	t.Run("Existent Persistent Volume within the namespace is gathered", func(t *testing.T) {
```

Comment:
Table-driven tests, please. Also, we should probably include the test for an error scenario.